R/prxgradl.R

```diff
@@ -66,11 +75,28 @@ cholpath <- function(X, lambdas = NULL, L =  diag(ncol(X)),
   	
     lambdas <- 10^seq(0, lambda.min.exp, length = 100)
   }
+  if (normalize){
+    S <- stats::cor(X)
+    D_scale <- sqrt(diag(stats::cov(X)))
+  }else{
+    S <- stats::cov(X)
+  }
   results <- list()
   for (i in 1:length(lambdas)){
-    results[[i]] <- prxgradchol(X, L, eps, 
-                              maxIter = maxIter, lambda = lambdas[i])
-    L <- results[[i]]$L
+    results[[i]] <- .Fortran("PRXGRD",as.integer(ncol(X)), as.double(S), 
```

Comment:
Why not use prxgradchol??

Reply:
- I wanted to be sure what it was doing. 
- I think it is more efficient as it avoids building the Sigma matrix before and then rescale it (little gain but still ..)

Reply:
In the future we could probably move everything to FORTRAN and just have R wrappers.